### PR TITLE
Bump SDK to 2.41.0

### PR DESCRIPTION
**Pull request by Que3216**

### CODE REVIEW

File: src/action.ts [hunk 1]
```diff
@@ -41,7 +39,7 @@ export const runMeticulousTestsAction = async (): Promise<void> => {
   // Init Sentry without sampling traces on the action run.
   // Children processes, (test run executions) will use
   // the global sample rate.
-  const sentryHub = await initSentry(1.0);
+  const sentryHub = await initSentry("report-diffs-action-v1", 1.0);
```
Comment (Que3216) [inline src/action.ts:42]: This is no longer the SDK version.

When we merge it into the Meticulous repo it'll all switch round a bit though -- we'll be using the git commit hash instead. I can then export that from the bundle (commitSha), so can specify the specific version here.

File: src/utils/logger.utils.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import { METICULOUS_LOGGER_NAME } from "@alwaysmeticulous/common";
+import log from "loglevel";
+
+export const initLogger: () => void = () => {
```
Comment (Que3216) [inline src/utils/logger.utils.ts:4]: These are exported from CLI, but think it's better just to copy and paste them and kill the CLI dep (not complex enough to be worth their own package)

File: src/utils/results-reporter.ts [hunk 3]
```diff
@@ -1,10 +1,18 @@
...
+  ExecuteTestRunResult,
+  RunningTestRunExecution,
+} from "@alwaysmeticulous/sdk-bundles-api";
 import { CodeChangeEvent } from "../types";
 import { updateStatusComment } from "./update-status-comment";
 
 const SHORT_SHA_LENGTH = 7;
 
+export interface EnrichedProject extends Project {
+  isGitHubIntegrationActive?: boolean;
```
Comment (alexivanov) [inline src/utils/results-reporter.ts:13]: I guess this will move to the underlying type in future versions of `@alwaysmeticulous/api`?

**Review by alexivanov — APPROVED** [review]
LGTM

File: src/utils/results-reporter.ts (hunk 3 above)
Comment (Que3216) [reply src/utils/results-reporter.ts:13]: Yep! I'll add it in an SDK PR.

**Pull request merged**